client/packages/system/src/Patient/Components/PatientOptionRenderer/PatientOptionRenderer.tsx

```diff
@@ -8,23 +8,25 @@ import {
 import { SearchInputPatient } from '../../utils';
 
 export const getPatientOptionRenderer =
-  (): AutocompleteOptionRenderer<SearchInputPatient> => (props, item) =>
-    (
-      <DefaultAutocompleteItemOption {...props} key={item.id}>
-        <Box display="flex" alignItems="flex-end" gap={1} height={25}>
-          <Box display="flex" flexDirection="row" gap={1} width={110}>
-            <Typography
-              overflow="hidden"
-              fontWeight="bold"
-              textOverflow="ellipsis"
-              sx={{
-                whiteSpace: 'nowrap',
-              }}
-            >
-              {item.code}
-            </Typography>
-          </Box>
-          <Typography>{item.name}</Typography>
+  (): AutocompleteOptionRenderer<SearchInputPatient> => (props, item) => (
+    <DefaultAutocompleteItemOption {...props} key={item.id}>
+      <Box display="flex" alignItems="flex-end" gap={1} height={25}>
+        <Box display="flex" flexDirection="row" gap={1} width={110}>
+          <Typography
+            overflow="hidden"
+            fontWeight="bold"
+            textOverflow="ellipsis"
+            sx={{
+              whiteSpace: 'nowrap',
+            }}
+          >
+            {item.code}
+          </Typography>
         </Box>
-      </DefaultAutocompleteItemOption>
-    );
+        <Box display="flex" flexDirection="row" gap={1} width={80}>
```

Comment:
on my screen, the 80px width isn't enough:

<img width="535" alt="Screenshot 2023-09-11 at 9 14 33 PM" src="https://github.com/openmsupply/open-msupply/assets/9192912/79410364-8093-4637-a9ae-632bb4e68ad3">

the rows where the dob is `30` do have the full date of birth, but it's wrapping over two rows. Either stretching the width or adding `whitespace: nowrap` would fix